src/Bundle/DependencyInjection/Driver/AbstractDriver.php

```diff
@@ -60,13 +61,15 @@ protected function setClassesParameters(ContainerBuilder $container, MetadataInt
 
     protected function addController(ContainerBuilder $container, MetadataInterface $metadata): void
     {
+        $viewHandler = new Reference('sylius.resource_controller.view_handler', ContainerInterface::NULL_ON_INVALID_REFERENCE);
+
         $definition = new Definition($metadata->getClass('controller'));
         $definition
             ->setPublic(true)
             ->setArguments([
                 $this->getMetadataDefinition($metadata),
                 new Reference('sylius.resource_controller.request_configuration_factory'),
-                new Reference('sylius.resource_controller.view_handler'),
+                $viewHandler,
```

Comment:
```suggestion
                new Reference('sylius.resource_controller.view_handler', ContainerInterface::NULL_ON_INVALID_REFERENCE),
```

I guess we can bring it inline now, so the only change would be adding the second argument to `Reference` constructor.